cdap-docs/developers-manual/source/data-exploration/filesets.rst

```diff
@@ -43,8 +44,12 @@ For example, in the configure method of your application::
       .setTableProperty("avro.schema.literal", schema.toString())
       .build());
 
-These dataset properties map directly to table properties in Hive. 
-For example, the dataset above would result in the following "create table" statement being generated::
+These dataset properties map directly to table properties in Hive. In the case of the
+``setBasePath`` method, the partial-path given is added to 
+``/<CDAP-home>/namespaces/<current-namespace>/data/``.
```

Comment:
It may not be proper to have a leading ``/`` here. I think ``<CDAP-home>`` would already be an absolute path starting with a ``/``